hadoop-common-project/hadoop-common/src/site/markdown/filesystem/filesystem.md

```diff
@@ -826,9 +826,9 @@ any successors.
 
 #### Postconditions
 
-    result = FSDataInputStream(0, FS.Files[p])
+    result = FSDataInputStream(0, FS'.Files[p])
 
-The result provides access to the byte array defined by `FS.Files[p]`; whether that
+The result provides access to the byte array defined by `FS'.Files[p]`; whether that
```

Comment:
open() is not mutating; FS` = FS; therefore FS is all that is needed

Reply:
Aha, I then misinterpreted 

> The suffix “’” (single quote) is used as the convention to indicate the state of the system after an operation

on https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-common/filesystem/notation.html

I'll add a small clarification there, and revert this.